src/components/Tokens/TokenTable/TokenRow.tsx

```diff
@@ -41,24 +41,33 @@ const Cell = styled.div`
   align-items: center;
   justify-content: center;
 `
-const StyledTokenRow = styled.div<{ first?: boolean; last?: boolean }>`
-  width: 100%;
-  height: 60px;
+const StyledTokenRow = styled.div<{ first?: boolean; last?: boolean; loading?: boolean }>`
+  background-color: transparent;
   display: grid;
-  grid-template-columns: 1fr 7fr 4fr 4fr 4fr 4fr 5fr 1.2fr;
   font-size: 15px;
+  grid-template-columns: 1fr 7fr 4fr 4fr 4fr 4fr 5fr 1.2fr;
+  height: 60px;
   line-height: 24px;
-
   max-width: ${MAX_WIDTH_MEDIA_BREAKPOINT};
   min-width: 390px;
   padding-top: ${({ first }) => (first ? '4px' : '0px')};
   padding-bottom: ${({ last }) => (last ? '4px' : '0px')};
   padding-left: 12px;
   padding-right: 12px;
+  transition: background-color 250ms ease;
```

Comment:
there were some transition variables added to theme so please use those

Reply:
ah cool didn't think those'd be in so soon